plutus-ledger-api/testlib/PlutusLedgerApi/Test/EvaluationEvent.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE DeriveAnyClass #-}
+
+module PlutusLedgerApi.Test.EvaluationEvent (
```

Comment:
Might as well write the `checkEvaluationEvent` function while you're here?

Also maybe we should have a couple of very simple synthetic tests using e.g. the scripts in `Examples` just so we can check the machinery works including e.g. the cost model params.

Reply:
> Might as well write the checkEvaluationEvent function while you're here?

This is in the next PR I'm about to open.

> Also maybe we should have a couple of very simple synthetic tests

Let me merge this PR and get back to this later, so that I can open the next one.